### PR TITLE
Re-import Sri Dasam Granth English Translations

### PR DESCRIPTION
**Pull request by Sarabveer**
Differences:
- Replace `“` and `”` with `"`
- Replace `‘` and `’` with `'`
- Split translations across 2 lines for Benti Choupai Sahib
- Small typo corrections
- Correct duplicate translations (same translation for 2 lines, rare case)

### CODE REVIEW

File: data/Sri Dasam Granth/0093.json [hunk 1]
```diff
@@ -210,7 +210,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "“I shall kill the fierce Chandi thus going in front of her just as the lion knocks down a goat in the forest.173.,",
+              "translation": "\"I shall kill the fierce Chandi thus going in front of her just as the lion knocks down a goat in the forest.173.,",
```
Comment (bhajneet) [inline data/Sri Dasam Granth/0093.json]: Ends in a comma and no closing quote. Next line is "DOHRA,".

File: data/Sri Dasam Granth/0100.json [hunk 2]
```diff
@@ -1285,7 +1285,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "The Lord is One and the Victory is of the True Guru.",
+              "translation": "",
```
Comment (bhajneet) [inline data/Sri Dasam Granth/0100.json]: ?

File: data/Sri Dasam Granth/0119.json [hunk 3]
```diff
@@ -633,7 +633,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "The Lord is One and the Victory is of the True Guru.",
+              "translation": "The Lord is one and the Victory is of the Lord.",
```
Comment (bhajneet) [inline data/Sri Dasam Granth/0119.json]: Can we check original?

File: data/Sri Dasam Granth/0123.json [hunk 4]
```diff
@@ -798,7 +798,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "“Lochan Dhum has been killed alongwith his soldiers",
+              "translation": "\"Lochan Dhum has been killed alongwith his soldiers",
```
Comment (bhajneet) [inline data/Sri Dasam Granth/0123.json]: The next few lines keep opening quotes and not ending them it seems

File: data/Sri Dasam Granth/0038.json [hunk 5]
```diff
@@ -523,7 +523,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "His net of karmas (actions) is invisibly spread.",
+              "translation": "His net of karmas (actions) is invisibly spread 6. 272",
```
[the remark this answers is not in the document]
Comment (Sarabveer) [reply data/Sri Dasam Granth/0038.json]: Most likely from the re-import. I just checked the `gurmukhi` and it doesn't have them.

File: data/Sri Dasam Granth/0033.json [hunk 6]
```diff
@@ -293,7 +293,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "The gods Shiva, Indra and Brahma repeat ‘Neti, Neti’ about Him.3.183.",
+              "translation": "The gods Shiva, Indra and Brahma repeat 'Neti, Neti' about Him.3.183.",
```
[the remark this answers is not in the document]
Comment (Sarabveer) [reply data/Sri Dasam Granth/0033.json]: Yea, this English translation is pretty odd, not standard anywhere.

File: data/Sri Dasam Granth/0050.json [hunk 7]
```diff
@@ -63,7 +63,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "Hearing the thunder of minghty warriors, the clouds felt shy.",
+              "translation": "Hearing the thunder of mighty warriors, the clouds felt shy.",
```
[the remark this answers is not in the document]
Comment (Sarabveer) [reply data/Sri Dasam Granth/0050.json]: I guess there were done manually. Most likely they were corrected on searchgurbani.com

**Comment by Sarabveer** [comment]
If anyone can get a physical or digital copy of this, that would be great.
image

**Comment by bhajneet** [comment]
Requires physical proof to reference

**Comment by Sarabveer** [comment]
Closed in favor of #1392

**Pull request closed without merging**